### ISSUE TITLE
Entering Altitude causes problems: "No usable PokeStops found in your area. Is your maximum distance too small?"

### ISSUE THREAD

**Issue by InfoClassified** [NecronomiconCoding/NecroBot#655]
I've always entered the altitude of the city I'm using in the bot in case Niantic does altitude checks alongside of long/lat GPS checks and entering the altitude in the most recent version causes the bot to be unable to find any stops:

"No usable PokeStops found in your area. Is your maximum distance too small?"

I imagine that is because using the Z-access for altitude makes the character "float" very high in the air compared to the bot's default altitude of 10.

**Comment by ShawnHardwick** [comment]
Do you still have this issue? Was your solution to just use the default altitude? I don't know if Niantic uses altitude at all and NecroBots behavior with the new altitude.

**Comment by InfoClassified** [comment]
Yeah, the issue is still there so I'm just using the default altitude. I'm not sure if Niantic even looks at altitude but I have always just changed it to whatever altitude the city I'm in is as an extra safety measure.

**Comment by ShawnHardwick** [comment]
Ok, I will keep this open as a bug and see if anyone else chimes in.

**Comment by p0c007** [comment]
Seems the issue is back again,  irrespective of what the altitude or location is, it says the same thing.  worked this morning, but not since this evening.

**Comment by Banno69** [comment]
I'm also getting the same issue. Seems to have stopped working around 9pm GMT

**Comment by GreenGoblinz** [comment]
I have this issue, when I just updated to latest version.
I think the problem is the Longitude is being recorded as Latitude because 
when I was reinstalling and entering coordinates it came out like this.
![pkmngo](https://cloud.githubusercontent.com/assets/20825040/17382737/075ee04e-5998-11e6-8fb0-50cf32acc4ae.jpg)

Notice it wrote Latitude accepted for Longitude.

**Comment by hongyili93** [comment]
i dont think that is the cause since even if u change the config.json afterward, same bug still exist

**Comment by trevorantram** [comment]
yeah it was working earlier but is down now does anyone know when it will come back up

**Comment by GreenGoblinz** [comment]
@hongyili93 Yeah you're right, I've tried changing the config file and that did not fix the bug.. well if anything I guess I found a syntax error.

**Comment by GreenGoblinz** [comment]
No clue trev, I wish someone would let us know the developers are aware of this bug and are working on it.. I had just popped a lucky egg when it stopped working.

**Comment by Wenkai186** [comment]
I have no idea what happened, it worked very well half hour ago... . Anyone have a solution ?
